include/irr/asset/IMeshBuffer.h

```diff
@@ -137,6 +144,10 @@ class IMeshBuffer : public virtual core::IReferenceCounted
     {
         return m_pipeline.get();
     }
+    inline const PipelineType* getPipeline_const() const
```

Comment:
why `_const` ?

Reply:
no idea why but below code
```
const ICPUMeshBuffer* mb = ...;
mb->getPipeline()->some_const_member_func();
```
tries to call non-const overload and gives me compile error. So this is kinda dirty workaround but idk whats going on

Reply:
do
`const_cast<const asset::IRenderpassIndependentPipeline*>(mb->getPipeline())`

Reply:
nah sorry its just `mb->getPipeline()` and the error is exactly 

>'irr::asset::ICPURenderpassIndependentPipeline *irr::asset::ICPUMeshBuffer::getPipeline(void)': cannot convert 'this' pointer from 'const irr::asset::ICPUMeshBuffer' to 'irr::asset::ICPUMeshBuffer &

whatever, i just made a const overload in ICPUMesBuffer overlapping the one from IMeshBuffer<....>. still dirty but at least theres no weird `getPipeline_const` now

Reply:
> whatever, i just made a const overload in ICPUMesBuffer overlapping the one from IMeshBuffer<....>. still dirty but at least theres no weird getPipeline_const now

Clean enough for me.